### PR TITLE
Small fixes to PGD overview page.

### PR DESCRIPTION
**Pull request by jpe442**

### CODE REVIEW

File: product_docs/docs/pgd/5/overview/index.mdx [hunk 1]
```diff
@@ -23,7 +23,7 @@ Each node (database) participating in a PGD group both receives changes from oth
 
 This is distinct from hot or warm standby, where only one master server accepts writes and all the other nodes are standbys that replicate either from the master or from another standby.
 
-You don't have to write to all the masters all of the time. A frequent configuration directs writes mostly to just one master called the [write leader](../terminology/#write-leader).
+You don't have to write to all the masters all of the time. A frequent configuration directly writes mostly to just one master called the [write leader](../terminology/#write-leader).
```
Comment (djw-m) [inline product_docs/docs/pgd/5/overview/index.mdx:26]: No, that should be directs as in directs traffic.

File: product_docs/docs/pgd/5/overview/index.mdx [hunk 2]
```diff
@@ -57,7 +57,7 @@ Each master node can be protected by one or more standby nodes, so any node that
 
 Replication continues between currently connected nodes even if one or more nodes are currently unavailable. When the node recovers, replication can restart from where it left off without missing any changes.
 
-Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for major versions of database software.
+Nodes can run different release levels, negotiating the required protocols to communicate. As a result, [EDB Postgres Distributed clusters can use rolling upgrades, even for major versions of database software](https://www.enterprisedb.com/docs/pgd/latest/upgrades/upgrading_major_rolling/).
```
Comment (djw-m) [inline product_docs/docs/pgd/5/overview/index.mdx:60]: That link text is way too big

File: product_docs/docs/pgd/5/overview/index.mdx [hunk 3]
```diff
@@ -75,7 +75,7 @@ Tables are created across both groups, so any change goes to all nodes, not just
 
 One node in each group is selected as the group's write leader. The write leader is the target for the main application's writes and queries. All other nodes are described as shadow nodes (or "read-write replica"), waiting to take over when needed. If a node loses contact, processing switches immediately to a shadow node to continue processing. If a group fails, processing can switch to the other group. Scalability isn't the goal of this architecture.
 
-Since writes are mainly to only one node, the possibility of contention between nodes is reduced to almost zero. As a result, performance impact is much reduced.
+Since writes are mainly to only one node, the possibility of contention between nodes is reduced to almost zero. As a result, performance impact is significantly reduced.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/overview/index.mdx:78]: Greatly... people read significantly as indicating magnitudes

**Review by djw-m — COMMENTED** [review]
Needs some tweaking

File: product_docs/docs/pgd/5/overview/index.mdx (hunk 1 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx:26]: oh jeez, I just got that read. Apologies!

File: product_docs/docs/pgd/5/overview/index.mdx (hunk 3 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx:78]: got it. 👍

File: product_docs/docs/pgd/5/overview/index.mdx (hunk 2 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx:60]: I will scale it back some.

File: product_docs/docs/pgd/5/overview/index.mdx (hunk 1 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx]: I changed it back.

File: product_docs/docs/pgd/5/overview/index.mdx [hunk 4]
```diff
@@ -57,7 +57,7 @@ Each master node can be protected by one or more standby nodes, so any node that
 
 Replication continues between currently connected nodes even if one or more nodes are currently unavailable. When the node recovers, replication can restart from where it left off without missing any changes.
 
-Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for [major versions of database software](https://www.enterprisedb.com/docs/pgd/latest/upgrades/upgrading_major_rolling/).
+Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for major versions of database software.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/overview/index.mdx:60]: At least conver it to relative

**Review by djw-m — COMMENTED** [review]
May want to loop this around to the PGD devs because with read-only nodes, some claims may need clarification.

File: product_docs/docs/pgd/5/overview/index.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx:60]: That is weird. I didn't think I had done so.

[the remark this answers is not in the document]
Comment (jpe442) [reply product_docs/docs/pgd/5/overview/index.mdx]: I added it back.

File: product_docs/docs/pgd/5/overview/index.mdx [hunk 5]
```diff
@@ -57,23 +57,23 @@ Each master node can be protected by one or more standby nodes, so any node that
 
 Replication continues between currently connected nodes even if one or more nodes are currently unavailable. When the node recovers, replication can restart from where it left off without missing any changes.
 
-Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for major versions of database software.
+Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for [major versions](https://www.enterprisedb.com/docs/pgd/latest/upgrades/upgrading_major_rolling/) of database software.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/overview/index.mdx:60]:
```suggestion
Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for [major versions](../upgrades/upgrading_major_rolling/) of database software.
```

**Review by djw-m — APPROVED** [review]
Make that link relative and you're LGTM

**Pull request merged**